operations/patch_util.go

```diff
@@ -451,23 +453,21 @@ func gitBranch() (string, error) {
 	return gitCmd("rev-parse", args...)
 }
 
-func getDefaultDescription() string {
+func getDefaultDescription() (string, error) {
 	desc, err := gitLastCommitMessage()
 	if err != nil {
-		grip.Debug("Couldn't create patch description using commit messages: " + err.Error())
+		return "", errors.Wrap(err, "Couldn't get last commit message")
 	}
 	branch, err := gitBranch()
 	if err != nil {
-		grip.Debug("Couldn't add branch to patch description: " + err.Error())
+		return "", errors.Wrap(err, "Couldn't get branch name")
 	}
+
 	branch = strings.TrimSpace(branch)
-	if desc == "" {
-		return branch
-	}
-	if branch == "" || strings.HasPrefix(desc, branch) {
-		return desc
+	if strings.HasPrefix(desc, branch) {
```

Comment:
Can branch/desc not be empty for sure?

Reply:
I couldn't think of a case where they'd be be empty that wasn't an error, aside from patching the initial commit in a repo.